lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -82,6 +87,14 @@ class MysqliStatement implements \IteratorAggregate, Statement
      */
     protected $_values = [];
 
+    /**
+     * Contains values from bindValue() that need to be sent
+     * using send_long_data *after* bind_param has been called.
```

Comment:
After https://github.com/doctrine/dbal/pull/3217/files#r219689244 is addressed, this will no longer be true. It will contain the values which we bound as LOB, not necessarily will be sent via `send_long_data()`.